docs/site/translation.md

```diff
@@ -102,6 +102,22 @@ Seven new emoji have been added (images above). These were released in Unicode 1
 
 As new locales reach Basic Coverage, their language names have been added for locales targeting modern coverage: Anii, Kuvi, …, Zhuang
 
+### DateTime formats
+
+There is a new “-relative” variant for [Date-Time Combined Formats](/translation/date-time/date-time-patterns#date-time-combined-formats).
+
+Before CLDR 48, there were two variants:
+- A “standard” variant for combining date with time, typically without literal text. In English this was “{1}, {0}”
+and resulted in combined date patterns like “March 20, 3:00 PM“, “March 20, 3:00-5:00 PM”, “tomorrow, 3:00 PM”, “tomorrow, 3:00-5:00 PM”.
```

Comment:
Could this also result in 3 days ago, 3 months ago? if so I think we should add a third example.

Reply:
Thanks, fixed in commit 2.